tcti/tcti_device.c

```diff
@@ -251,8 +250,6 @@ TSS2_RC InitDeviceTcti (
     }
     else
     {
-        (*tpmLocalTpmPrintf)(NO_PREFIX, "Initializing %s Interface\n", interfaceName );
```

Comment:
There's a call to TCTI_LOG after the data is populated which is effectively the same (see line 270). The only thing that happens between the call that's removed and the call that's added are a few assignments. You're concerned that something will go wrong between the two locations and you won't get the debug message?